lib/web_ui/lib/src/engine/history/history.dart

```diff
@@ -22,80 +22,59 @@ part of engine;
 ///  * [MultiEntriesBrowserHistory]: which creates a set of states that records
 ///    the navigating events happened in the framework.
 abstract class BrowserHistory {
+  static BrowserHistory defaultImpl({required UrlStrategy? urlStrategy}) {
```

Comment:
Avoid abbreviations: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations

But also, neither "default" nor "implementation" are descriptive. I'd just call it `multiEntry`.